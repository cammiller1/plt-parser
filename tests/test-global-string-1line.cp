string s = "declared and intialized in same line";
print(s);