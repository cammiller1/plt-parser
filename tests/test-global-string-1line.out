declared and intialized in same line